python/schemas/v2/Mapping.schema

```diff
@@ -180,6 +180,9 @@
                 "per_try_timeout": {
                     "type": "string"
                 },
+                "timeout_ms": {
+                    "type": "integer"
+                },
```

Comment:
It looks like you didn't add `respect_dns_ttl` to `pkg/api/getambassador.io/v3alpha1/mapping_types.go`?